docs/tutorials/serverless/cpu/run-ollama-inference.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Run an Ollama Server on a RunPod CPU
+description: Learn to set up and run an Ollama server on RunPod CPU for inference with this step-by-step tutorial.
+---
+
+In this guide, you will learn how to run an Ollama server on your RunPod CPU for inference.
+By the end of this tutorial, you will have a fully functioning Ollama server ready to handle requests.
+
+## Setting up your Endpoint
+
+:::note
+
+Use a Network volume to attach to your Worker so that it can cache the LLM and decrease cold start times.
+
+:::
+
+To begin, you need to set up a new endpoint on RunPod.
+
+1. Log in to your [RunPod account](https://www.runpod.io/console/console/home).
+2. Navigate to the **Serverless** section and select **New Endpoint**.
+3. Choose **CPU** and provide a name for your Endpoint.
+4. Configure your Worker settings according to your needs.
+5. In the **Container Image** field, enter the `pooyaharatian/runpod-ollama:0.0.7` container image.
+6. In the **Container Start Command** field, specify the Ollama supported model, such as `orca-mini`.
```

Comment:
maybe throw in a link here on Ollama supported model, https://ollama.com/library . and also let the user know they have to be conscious that inference on cpu is way slower so they'll need to select a smaller model.